WordPress/gutenberg
# Site Editor: Add keyboard shortcut help modal

## Description
Part of #21245.

PR adds keyboard shortcuts help modal to the "More tools and options" dropdown.

## How has this been tested?
1. Go to Appearance > Editor
2. Open the "More tools and options" dropdown
3. Click on the "Keyboard shortcuts" menu item or use <kbd>CTRL</kbd>+<kbd>Option/Alt</kbd>+<kbd>H</kbd>
4. This should open new help modal

## Screenshots 
![CleanShot 2021-12-29 at 13 36 37](https://user-images.githubusercontent.com/240569/147648397-098d094e-ea70-47ff-8931-633ab47d493c.png)

## Types of changes
Enhancement

## Checklist:
